decidim-meetings/app/commands/decidim/meetings/join_meeting.rb

```diff
@@ -50,22 +51,38 @@ def save_registration_form
       end
 
       def create_registration
-        @registration = Decidim::Meetings::Registration.create!(meeting: meeting, user: user)
+        Decidim::Meetings::Registration.create!(meeting: meeting, user: user)
       end
 
       def can_join_meeting?
         meeting.registrations_enabled? && meeting.has_available_slots?
       end
 
       def send_email_confirmation
-        Decidim::Meetings::RegistrationMailer.confirmation(user, meeting, registration).deliver_later
+        Decidim::Meetings::RegistrationMailer.confirmation(
+          @user,
+          @meeting,
+          Decidim::Meetings::Registration.last
+        ).deliver_now
+      end
+
+      def send_notification_confirmation
+        Decidim::EventsManager.publish(
+          event: "decidim.events.meetings.meeting_registration_confirmed",
+          event_class: Decidim::Meetings::MeetingRegistrationNotificationEvent,
+          resource: @meeting,
+          recipient_ids: [@user.id],
+          extra: {
+            registration_code: Decidim::Meetings::Registration.last.code
```

Comment:
This might not be the real code for the given user, we might be getting in a race condition here. Please look for the registration belonging to the given user and meeting. You can find it if you undo the changes in line 53 and use `@registration.code` here

Reply:
Thanks very much